### PR TITLE
Add overbright clamp option to gfx menu

### PR DESCRIPTION
**Pull request by slipher**
Needs https://github.com/DaemonEngine/Daemon/pull/1364 or https://github.com/DaemonEngine/Daemon/pull/1350.

### CODE REVIEW

File: pkg/unvanquished_src.dpkdir/ui/options_graphics.rml [hunk 1]
```diff
@@ -99,6 +99,11 @@
 					</select>
 					<h3><translate>Lighting system</translate></h3>
 				</row>
+				<row>
+					<h3><translate>Brightness clamping</translate></h3>
+					<input cvar="r_overbrightDefaultClamp" type="checkbox" />
+					<p><translate>Limit max brightness of map lighting of world surfaces and models (better compat with majority of legacy maps)</translate></p>
```
Comment (illwieckz) [inline pkg/unvanquished_src.dpkdir/ui/options_graphics.rml:105]: It's better to not use abbreviations to not make the translation harder to do. Anyway I recommend to use instead something like “`Old brightness limit of map lighting of world surfaces and models (the way legacy maps looked before)`” as it is not really a matter of data compatibility but more a matter of human habituation.

**Comment by illwieckz** [comment]
LGTM % the comment.

**Comment by illwieckz** [comment]
Actually I can commit the suggested easier-to-translate string myself. Let's merge this and start a build of RC3

File: pkg/unvanquished_src.dpkdir/ui/options_graphics.rml (hunk 1 above)
Comment (VReaperV) [reply pkg/unvanquished_src.dpkdir/ui/options_graphics.rml:105]: Something like "LDR lighting" might be more concise.

**Pull request merged**